src/hrc721.ts

```diff
@@ -62,7 +62,19 @@ export class HRC721 extends BaseHRC721 {
   }
 
   async getApproved(tokenId: string): Promise<string> {
-    return ''
+    if (!tokenId) {
+      throw new Error('You must to provide a tokenId')
```

Comment:
same as above `You must provide a tokenId`

Reply:
Got it!